### ISSUE TITLE
Error with lsGetResponses

### ISSUE THREAD

**Issue by jsibley2** [kalimu/LimeRick#1]
I can get a session key and lsGetSummary works without a problem, showing 8 completed responses.
However, lsGetResponses(surveyID, completionStatus = 'complete') or just lsGetResponses(surveyID) give the following error:

Error in lsAPI(method = method, params = params, lsAPIurl = lsAPIurl) : 
  Server is responding but not in a proper way. Please check the API URL.

Am I missing a necessary parameter or do you have another ID of what the problem might be?

Thanks.

**Comment by kalimu** [comment]
That is strange indeed. Please check if your LimeSurvey user has access to the responses of the survey. Perhaps I could help you more, but you need to provide some code for replication (I can't replicate this issue on my surveys). You can contact me by mail as well (kamil.wais@gmail.com).

**Comment by r0bis** [comment]
Yup, I have exact same problem. I can get responses with `limer` but not with `limeRick` and I get the same kind of error message from `limerick`. I am sure the URL is correct. With respect to survey permissions - the user I am logging in has permissions to read responses, not anything else. The user can list surveys that he has permission to read responses from, but not get the results. Before I gave the read permission, the user could not list any surveys - response was empty.
